FRE/fre_make.json

```diff
@@ -191,7 +191,11 @@
                 },
                 "RUNenv": {
                     "description": "Commands needed at the beginning of a RUN in dockerfile",
-                    "type": ["array"],
+                    "anyOf": [
+                      {"type": "array"}, 
+                      {"type": "string"}
+                    ],
+                    "type": ["array", "string"],
                     "items": {"type": "string"}
```

Comment:
```
"items": {"type": "string"}
```
This would make more sense inside the `"type": "array"` subschema, since it's not applicable in the case where RUNenv is a string.